backend/entry/serializers.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework import serializers
+from user_resource.serializers import UserResourceSerializer
+from .models import (
+    Entry, Attribute, FilterData, ExportData
+)
+
+
+class EntrySerializer(UserResourceSerializer):
+    """
+    Entry Model Serializer
+    """
+    class Meta:
+        model = Entry
+        fields = ('__all__')
+
+
+class AttributeSerializer(serializers.ModelSerializer):
+    """
+    Entry Attribute Model Serializer
+    """
+
+    class Meta:
+        model = Attribute
+        fields = ('__all__')
+
+    # Validations
+    def validate_region(self, attribute):
```

Comment:
As in analysis framework, here we need to have `validate_entry(self, entry)`

Also, raise error 'Invalid Entry'